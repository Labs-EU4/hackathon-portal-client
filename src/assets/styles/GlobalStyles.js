import { createGlobalStyle } from "styled-components";

import bg from '../images/Matrix-bg.jpg';

export const GlobalStyles = createGlobalStyle`
  /*MEYER RESET*/
  html, body, div, span, applet, object, iframe,
  h1, h2, h3, h4, h5, h6, p, blockquote, pre,
  a, abbr, acronym, address, big, cite, code,
  del, dfn, em, img, ins, kbd, q, s, samp,
  small, strike, strong, sub, sup, tt, var,
  b, u, i, center,
  dl, dt, dd, ol, ul, li,
  fieldset, form, label, legend,
  table, caption, tbody, tfoot, thead, tr, th, td,
  article, aside, canvas, details, embed,
  figure, figcaption, footer, header, hgroup,
  menu, nav, output, ruby, section, summary,
  time, mark, audio, video {
      margin: 0;
      padding: 0;
      border: 0;
      font-size: 100%;
      font: inherit;
      vertical-align: baseline;
  }

  /* make sure to set some focus styles for accessibility */
  :focus {
      outline: 0;
  }

  /* HTML5 display-role reset for older browsers */
  article, aside, details, figcaption, figure,
  footer, header, hgroup, menu, nav, section {
      display: block;
  }

  body {
      line-height: 1;
  }

  ol, ul {
      list-style: none;
  }

  blockquote, q {
      quotes: none;
  }

  blockquote:before, blockquote:after,
  q:before, q:after {
      content: '';
      content: none;
  }

  table {
      border-collapse: collapse;
      border-spacing: 0;
  }

  input[type=search]::-webkit-search-cancel-button,
  input[type=search]::-webkit-search-decoration,
  input[type=search]::-webkit-search-results-button,
  input[type=search]::-webkit-search-results-decoration {
      -webkit-appearance: none;
      -moz-appearance: none;
  }

  input[type=search] {
      -webkit-appearance: none;
      -moz-appearance: none;
      -webkit-box-sizing: content-box;
      -moz-box-sizing: content-box;
      box-sizing: content-box;
  }

  textarea {
      overflow: auto;
      vertical-align: top;
      resize: vertical;
  }

  audio,
  canvas,
  video {
      display: inline-block;
      *display: inline;
      *zoom: 1;
      max-width: 100%;
  }

  audio:not([controls]) {
      display: none;
      height: 0;
  }

  [hidden] {
      display: none;
  }

  html {
      font-size: 100%; /* 1 */
      -webkit-text-size-adjust: 100%; /* 2 */
      -ms-text-size-adjust: 100%; /* 2 */
  }

  a {
      -webkit-tap-highlight-color: transparent;
  }

  a:focus {
      outline: none;
  }

  a:active,
  a:hover {
      outline: 0;
  }

  img {
      border: 0; /* 1 */
      -ms-interpolation-mode: bicubic; /* 2 */
  }

  figure {
      margin: 0;
  }

  form {
      margin: 0;
  }

  fieldset {
      border: 1px solid #c0c0c0;
      margin: 0 2px;
      padding: 0.35em 0.625em 0.75em;
  }

  legend {
      border: 0; /* 1 */
      padding: 0;
      white-space: normal; /* 2 */
      *margin-left: -7px; /* 3 */
  }

  button,
  input,
  select,
  textarea {
      font-size: 100%; /* 1 */
      margin: 0; /* 2 */
      vertical-align: baseline; /* 3 */
      *vertical-align: middle; /* 3 */
  }

  button,
  input {
      line-height: normal;
      -webkit-tap-highlight-color: transparent;
  }

  button,
  select {
      text-transform: none;
  }

  button,
  html input[type="button"], /* 1 */
  input[type="reset"],
  input[type="submit"] {
      -webkit-appearance: button; /* 2 */
      cursor: pointer; /* 3 */
      *overflow: visible;  /* 4 */
  }

  button[disabled],
  html input[disabled] {
      cursor: default;
  }

  input[type="checkbox"],
  input[type="radio"] {
      box-sizing: border-box; /* 1 */
      padding: 0; /* 2 */
      *height: 13px; /* 3 */
      *width: 13px; /* 3 */
  }

  input[type="search"] {
      -webkit-appearance: textfield; /* 1 */
      -moz-box-sizing: content-box;
      -webkit-box-sizing: content-box; /* 2 */
      box-sizing: content-box;
  }

  input[type="search"]::-webkit-search-cancel-button,
  input[type="search"]::-webkit-search-decoration {
      -webkit-appearance: none;
  }

  button::-moz-focus-inner,
  input::-moz-focus-inner {
      border: 0px;
      padding: 0px;
      outline: 0px !important;
  }

  textarea {
      overflow: auto; /* 1 */
      vertical-align: top; /* 2 */
  }

  table {
      border-collapse: collapse;
      border-spacing: 0;
  }

  html,
  button,
  input,
  select,
  textarea {
      color: #222;
  }

  ::-moz-selection {
      background: #b3d4fc;
      text-shadow: none;
  }

  ::selection {
      background: #b3d4fc;
      text-shadow: none;
  }

  img {
      vertical-align: middle;
  }

  fieldset {
      border: 0;
      margin: 0;
      padding: 0;
  }

  textarea {
      resize: vertical;
  }

  .chromeframe {
      margin: 0.2em 0;
      background: #ccc;
      color: #000;
      padding: 0.2em 0;
  }

  /* Others ... */
  *, *::before, *::after {
      box-sizing: border-box;
  }

  :root {
      @import url('https://fonts.googleapis.com/css?family=Orbitron&display=swap');
      font-family: 'Orbitron', sans-serif !important;
      font-size: 62.5%;
      font-size: 57.5%;
      scroll-behavior: smooth;
      line-height: normal;
      --app-background: rgba(15,15,15,0.925);
      --app-text-color: #fff;
      --dash-side-margin: 20px;
      --app-padding: 21px;
      --border-radius: 5px;
      --border-color: rgba(255,255,255,.15);
      --border-color-low: rgba(255,255,255,.09);
      --header-height: 47px;
      --app-dropdown-bg: rgb(51,51,51);
      --app-message-bg: rgba(255,255,255,0.2);
      --icon-color: #fff;
      --a-show-full-duration: 0.3s;
      --a-show-full-curve: ease-out;
      --a-curve: ease;
      --a-fast: 0.15s;
      --a-default: all var(--a-fast) var(--a-curve);
  }

  body {
      max-width: 100vw;
      overflow: hidden;
      text-rendering: optimizeLegibility;
      font-weight: bold;
      -webkit-font-smoothing: antialiased;
      text-shadow: 0 1px 5px rgba(0,0,0,.1);
      line-height: 1.2;
      color: black;
      background-image: url(${bg}), linear-gradient(-225deg, #77FFD2 0%, #6297DB 48%, #1EECFF 100%);
      
      .dashboard-bg {
          height: 105vh;
          border: 3px solid red;
      }
  }

  .particles {
      position: fixed;
      top: 0;
      right: 0;
      bottom: 0;
      left: 0;
      z-index: 10;
  }
`;
