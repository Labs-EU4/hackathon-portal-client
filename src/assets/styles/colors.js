//COLORS
export const white = '#F9F9F9';
export const black = '#010038';

//RED COLOR
export const red = '#E34449';
export const brightRed = '#F34D21';

//YELLOW COLOR
export const dimYellow = '#ffdb00';

//GREY COLOR
export const grey = '#39444D';
export const textLightGrey = '#999';
export const textDarkGrey = '#666';
export const bgLightGrey = '#f0f0f0';
export const bgGrey = '#dfe3e6';
export const borderGrey = '#39444D';
// export const lightGrey = '#F5F5F5';

//GREEN COLOR
export const lightGreen = '#afeeee';
export const green = '#5aac44';
export const brightgreen = '#25d366';
export const darkGreen = '#1b9849';


// export const primary = '#04c8de'; //--> same as the above, good combo
// export const primary = '#F5F5F5'; //--> same as the above, good combo
// export const primary = '#FF3636'; //--> same as the above, good combo
// export const primary = '#47e4bb'; //--> same as the above, good combo

//PRIMARY COLOR
export const primaryLight = '#81DBFD';
export const primary = '#03AEEF';
export const primaryDark = '#03AEEF';
// export const primary = '#8CCAE9';
// export const primaryLight = '#04c8de';
// export const primary = '#40e0d0';

//SECONDARY COLOR
export const secondaryLight = '#F8C485';
export const secondary = '#f39422';
export const secondaryDark = '#DA7C0A';
// export const secondaryDark = '#F34D21';






export const backgroundBlack = '#222';
export const backgroundWhite = '#f4f4f4';
export const lightBlue = '#0ff';

// export const white = '#f4f7f6'; // --> GOOD WHITE
export const textWhite = '#F8F9FA';
export const primaryWhite = '#eff';
export const secondaryWhite = '#eee';
export const tertiaryWhite = '#ddd';
export const quaternaryWhite = '#ccc';
export const senaryWhite = '#aaa';




// export const black = '#000';
// export const black = '#1F1C05'; --> VERY GOOD BLACK SHADE
// export const black = '#020404 '; --> ALRIGHT BLACK!!!
// export const black = '#252525'; //--> SUPER GOOD BLACK
// export const black = '#202121';
export const softGreen = '#aec';
export const spaceBlue= '#03AEEF';
export const brightPurple= '#4444dd';
export const darkTwitterTheme = '#15202B';
export const lightDarkTwitterInputBg = '253341#';
export const fbCopyCatLightTheme = '#F3F4F6';
export const youTubeColorBlue = '#065FD4';
export const flickrBlue = '#1396E7';
export const btnBlue = '#0973FF';

//Add theme using styled components

//MAIN THEME COLORS
// export const primaryColor = '#BDE6EC';
// export const primaryColor = '#aec';
// export const primaryColor = '#ffdb00';
// export const primaryColor = '#6747C7'; //--> SUPER GOOD!!
// export const primaryColor = '#f8f4ff'; //--> GOOD WHITE VERSION!!
// export const primaryColor = '#fada5e'; //--> ROYAL YELLOW
// export const lightPrimaryColor = '#FEF33E'; //--> BRIGHT COOL YELLOW
// export const primaryColor = '#FFFF01'; //--> VERY BRIGHT YELLOW
// export const primaryColor = '#F4B75D'; //--> VERY BRIGHT YELLOW
// export const primaryColor = '#448ef6'; //--> VERY BRIGHT YELLOW
// export const primaryColor = '#6decb9'; //--> VERY BRIGHT YELLOW


// export const primaryColor = '#537ec5'; //--> super good color, mostly used theme!!!!
// export const turquoise = '#ff5edf'; //--> change to secondary color

export const tertiaryColor = '#5B5F97';
export const tertiaryColorDarker = '#484b77';

export const borderGrayTheme = '#39444D';

// export const primaryColor = '#1396E7'; //--> VERY GOOOD
// export const primaryColor = '#065FD4'; //--> VERY good blue, 2nd/3rd
// export const primaryColor = '#FFf'; //--> MOST USED YELLOW!!!! 
// export const primaryColor = '#B390EE';
// export const primaryColor = '#CC2D8A';
// export const primaryColor = '#46130B';
// export const primaryColor = '#FAD52D';

// export const turquoise = '#40e0d0 ';
// export const turquoise = '#4dd599'; --> VERY GOOD
// export const turquoise = '#47e4bb'; --> EVEN BETTER
// export const turquoise = '#3fc5f0';
//DARK-MODE TOGGLE COLORS
export const sunrise = '#ffc922';
export const yellowSun = '#ffdb00';
export const blueMoon = '#003366';
export const papayawhip = 'papayawhip';


// COOL COLORS 
export const newSecondaryColor = '#B390EE';
export const newTertiaryColor = '#B0EEEE';


// DARK MODE
export const MainBackgroundColor = '#29272F';
