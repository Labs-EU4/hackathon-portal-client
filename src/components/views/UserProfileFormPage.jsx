import React, { useEffect } from "react";
import { useDispatch, useSelector } from "react-redux";
import UserProfileForm from "../templates/UserProfileForm";
import { fetchUserProfile } from "../../store/user/actions";

const UserProfileFormPage = ({ isProfileOpen, setIsProfileOpen }) => {
  const dispatch = useDispatch();
  const { userId } = useSelector(state => state.currentUser);
  const userDetails = useSelector(state => state.currentUser);
  useEffect(() => {
    dispatch(fetchUserProfile(userId));
  }, [dispatch, userId]);

  return (
    <>
      <UserProfileForm 
        initialState={userDetails}
        {...{isProfileOpen}} 
        {...{setIsProfileOpen}} 
      />
    </>
  );
};

export default UserProfileFormPage;
