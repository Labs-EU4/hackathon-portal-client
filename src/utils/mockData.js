export const initialState = {
  events: {
    data: [
      {
        id: 1,
        event_title: "EuroHack",
        event_description: "the best event hack hack hack macbook",
        creator_id: 1,
        start_date: "10/09/11",
        end_date: "08/07/13",
        location: "Europe",
        guidelines: "cool hekhkdsjfhsdkjf skdjfhskdjfhkjsdfh skjdfhksjdfh",
        participation_type: "Team",
        category_id: 1,
        requirements: [
          {
            video_url:
              "https://res.cloudinary.com/ibkrice-com/video/upload/v1581259780/Harvesting_ifato8.mp4"
          }
        ],
        ProfileImage: [
          {
            image_url:
              "https://res.cloudinary.com/ibkrice-com/video/upload/v1581259780/Harvesting_ifato8.mp4"
          }
        ],
        rubrics: [
          "Presentation",
          "Product Fit",
          "Functionality",
          "Innovation",
          "Product Design",
          "Extensibility"
        ]
      },
      {
        id: 2,
        event_title: "World",
        event_description: "number one hacker event in the world",
        creator_id: 1,
        start_date: "07/09/11",
        end_date: "08/07/13",
        location: "Asia",
        guidelines: "come well prepared,no cheating,no fighting",
        participation_type: "Team",
        category_id: 2
      }
    ],
    categories: [
      { id: 2, category_name: "Summer Hackton" },
      { id: 3, category_name: "World Hackton" },
      { id: 3, category_name: "Asia Hackton" },
      { id: 3, category_name: "Gamer's Hackton" },
      { id: 3, category_name: "Advanced Hackton" }
    ]
  },

  currentUser: {
    token: 347534574536,
    userId: 1,
    fullname: "Jake Tom",
    email: "8omemail@google.com",
    username: "Jake22",
    bio: "I like coding ",
    image: ""
  },
  submissions: [
    {
      id: 1,
      project_title: "Mock Project",
      participant_or_team_name: "Mock Team",
      event_id: 7,
      video_url: "",
      git_url: "https://www.google.com/",
      project_writeups: "No mock write up",
      submitted_by: 14,
      created_at: "2020-03-02",
      updated_at: "2020-03-02",
      average_rating: 0,
      acted_judges: 0,
      number_of_judges: 0
    }
  ],

  teams: [
    { id: 1, team_name: "The A team", team_lead: 2 },
    { id: 2, team_name: "Hacking central", team_lead: 3 },
    { id: 3, team_name: "blackhats", team_lead: 4 },
    { id: 4, team_name: "Spies", team_lead: 1 }
  ],
  participant_team_members: [
    {
      team_member: 3,
      team_id: 1
    },
    {
      team_member: 4,
      team_id: 1
    },
    {
      team_member: 5,
      team_id: 1
    }
  ]
};
