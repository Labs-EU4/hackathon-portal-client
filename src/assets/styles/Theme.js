import * as color from './variables/colors';

const size = {
    mobileS: '320px',
    mobileM: '375px',
    mobileL: '425px',
    tablet: '768px',
    laptop: '1024px',
    laptopL: '1440px',
    desktop: '2560px'
};

export const theme = {
    color: {
        primary: {
            // light: color.primaryLight,
            regular: color.primary,
            // dark: color.primaryDark
        },
        secondary: {
            // light: color.secondaryLight,
            regular: color.secondary,
            // dark: color.secondaryDark
        },
        white: {
            regular: color.white,
        },
        black: {
            regular: color.black,
        },
        red: {
            regular: color.red,
            bright: color.brightRed
        },
        // grey: color.grey,
        grey: {
            regular: color.grey
        },
        green: {
            light: color.lightGreen,
            regular: color.green,
            dark: color.darkGreen
        },
    },
    shadow: {
        text: `text-shadow: .15px .15px 1px ${color.black}, .75px .5px 1.5px ${color.black}, .15px .15px 1px ${color.black};`,
        text_links: `1px .75px 1.5px ${color.black}`,
        box: `box-shadow: 0 3px 20px 0 rgba(0, 0, 0, .2)`,
        card: `box-shadow: 1px .5px 3px rgba(0, 0, 0, .15)`,
        intenseBox: `box-shadow: 0 2.5px 10px ${color.black};`,
        filter: `filter: drop-shadow(.5px .25px 1.5px ${color.black});
        filter: drop-shadow(.15px .15px 1px ${color.black}, .5px .25px 1.5px ${color.black}, .65px .4px 1px ${color.black});`,
        icon: `filter: drop-shadow(.5px .25px 1.5px ${color.black});
        filter: drop-shadow(25px .25px 10px ${color.black}, .5px .25px 1.5px ${color.black}, .65px .4px 1px ${color.black});`,
        btn: `2px 2px 3px ${color.black}`
    },
    flex: {
        center: 'display: flex; justify-content: center; align-items: center;',
        columnCenter: 'display: flex; flex-direction: column; justify-content: center; align-items: center;',
        column: 'display: flex; flex-direction: column;',

        custom: (justify='start', align='start', direction='row') => {
            return `
                display: flex; 
                flex-direction: ${direction}
                justify-content: ${justify}; 
                align-items: ${align};
            `
        },
    },
    device: {
        mobileS: `(min-width: ${size.mobileS})`,
        mobileM: `(min-width: ${size.mobileM})`,
        mobileL: `(min-width: ${size.mobileL})`,
        tablet: `(min-width: ${size.tablet})`,
        laptop: `(min-width: ${size.laptop})`,
        laptopL: `(min-width: ${size.laptopL})`,
        desktop: `(min-width: ${size.desktop})`,
        desktopL: `(min-width: ${size.desktop})`
    }
};