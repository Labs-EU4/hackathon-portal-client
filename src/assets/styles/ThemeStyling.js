import * as color from './variables/colors';

const size = {
    mobileS: '320px',
    mobileM: '375px',
    mobileL: '425px',
    tablet: '768px',
    laptop: '1024px',
    laptopL: '1440px',
    desktop: '2560px'
};

export const theme = {
    color: {
        primary: {
            light: '#81DBFD',
            regular: color.primary,
            // dark: color.primaryDark
        },
        secondary: {
            // light: color.secondaryLight,
            regular: color.secondary,
            // dark: color.secondaryDark
        },
        white: {
            regular: color.white,
            bg: color.bgWhite
        },
        black: {
            light: '#212121',
            regular: color.black,
        },
        red: {
            regular: color.red,
            bright: color.brightRed
        },
        blue: {
            light: color.lightBlue,
            regular: color.blue,
            bg: '#2196f3'
        },
        grey: {
            light: '#696969',
            regular: color.grey,
            dark: '#444',
            bg: color.bgGrey,
            border: 'rgba(14,19,24,.15)'
        },
        green: {
            light: color.lightGreen,
            regular: color.green,
            dark: color.darkGreen
        },
        success: color.success,
        warning: color.warning,
        danger: color.danger,
        info: color.info,
        link: {
            hover: 'rgba(14,19,24, .07)',
        }
    },
    fontSize: {
        smaller: "font-size: 1.2rem",
        small: "font-size: 1.4rem;",
        base: "font-size: 1.6rem;",
        medium: "font-size: 1.8rem",
        large: "font-size: 2rem",
        h1: "font-size: 7.5rem;",
        h2: "font-size: 5.5rem;",
        h3: "font-size: 3.5rem;",
        h4: "font-size: 2.5rem;",
        button: "font-size: 1.8rem;"
    },
    shadow: {
        text: `text-shadow: .5px .5px 1px ${color.black}`,
        text_links: `1px .75px 1.5px ${color.black}`,
        box: `box-shadow: 0 3px 20px 0 rgba(0, 0, 0, .2)`,
        card: `box-shadow: 1px .5px 3px rgba(0, 0, 0, .15)`,
        intenseBox: `box-shadow: 0 2.5px 10px ${color.black};`,
        filter: `filter: drop-shadow(.15px .15px 1.5px ${color.black});
        filter: drop-shadow(-.15px -.15px 1px ${color.black});`,
        btn: `2px 2px 3px ${color.black}`
    },
    flex: {
        center: 'display: flex; justify-content: center; align-items: center;',
        columnCenter: 'display: flex; flex-direction: column; justify-content: center; align-items: center;',
        column: 'display: flex; flex-direction: column;',

        custom: (justify='start', align='start', direction='row', wrap='no-wrap') => {
            return `
                display: flex; 
                flex-direction: ${direction}
                justify-content: ${justify}; 
                align-items: ${align};
                flex-wrap: ${wrap};
            `
        },
    },
    device: {
        mobileS: `(min-width: ${size.mobileS})`,
        mobileM: `(min-width: ${size.mobileM})`,
        mobileL: `(min-width: ${size.mobileL})`,
        tablet: `(min-width: ${size.tablet})`,
        laptop: `(min-width: ${size.laptop})`,
        laptopL: `(min-width: ${size.laptopL})`,
        desktop: `(min-width: ${size.desktop})`,
        desktopL: `(min-width: ${size.desktop})`
    }
};