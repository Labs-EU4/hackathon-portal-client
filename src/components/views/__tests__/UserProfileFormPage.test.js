import React from "react";
import { Router } from "react-router";
import { createMemoryHistory } from "history";
import { Provider } from "react-redux";
import { render } from "@testing-library/react";
import configureStore from "redux-mock-store";
import "@testing-library/jest-dom/extend-expect";
import { initialState } from "../../../utils/mockData";
import UserProfileFormPage from "../UserProfileFormPage";
import { theme } from "../../../assets/styles/ThemeStyling";
import { ThemeProvider } from "styled-components";

const history = createMemoryHistory();

let mockStore;
let store;
let component;

beforeEach(() => {
  mockStore = configureStore();
  store = mockStore(initialState);

  component = render(
    <Router history={history}>
      <Provider store={store}>
        <ThemeProvider theme={theme}>
          <UserProfileFormPage />
        </ThemeProvider>
      </Provider>
    </Router>
  );
});

describe("Component UserProfileFormPage.js renders properly", () => {
  it("asserts that the component renders properly", () => {
    expect(component).toMatchSnapshot();
  });
  // it("asserts that the first character of the user's email shows up on the menu", () => {
  //   expect(component.getByText("8")).toBeInTheDocument();
  // });
  it("asserts that the <p> Username text node renders properly on the form component", () => {
    expect(component.getByText("Jake22")).toBeInTheDocument();
  });
  it("asserts that the <textarea> placeholder for current User bio text node renders properly on the form component", () => {
    expect(component.queryByPlaceholderText("bio")).toBeInTheDocument();
  });

  it("asserts that the <textarea> text node, 'I like coding' for current User bio text node renders properly on the form component", () => {
    expect(component.queryByText("I like coding")).toBeInTheDocument();
  });
});
