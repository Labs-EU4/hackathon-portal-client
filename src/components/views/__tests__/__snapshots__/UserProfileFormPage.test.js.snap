// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component UserProfileFormPage.js renders properly asserts that the component renders properly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="sc-bdVaJa gOhKpA"
        >
          <div
            class="sc-htpNat dAVxPZ"
          >
            <a
              href="/dashboard"
            >
              <img
                alt="Hackton - Organise hackathons"
                class="sc-ifAKCX gCNDrT"
                src="Hackton-logo.png"
              />
            </a>
            <div
              class="sc-tilXH iUsghC"
            >
              <div
                class="sc-EHOje kXozTT"
              >
                8
                <ul
                  class="sc-bZQynM dfAYGq"
                >
                  <li>
                    <a
                      href="/dashboard/profile/edit"
                    >
                      Settings
                    </a>
                  </li>
                  <li>
                    <span>
                      LogOut
                    </span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-hzDkRC cBfjrB"
        >
          <div
            class="sc-jhAzac jxgZSd sc-jbKcbu iFCIT"
          >
            <div
              class="sc-dVhcbM iWcBSy"
            >
              <h3
                class="sc-jzJRlG dhdOHq"
              >
                Edit Profile
              </h3>
            </div>
            <div
              class="sc-fAjcbJ DOSjX"
            >
              <div
                class="sc-cJSrbW sc-ksYbfQ sc-jqCOkK jnpjEb"
              >
                <form
                  action="#"
                >
                  <label
                    class="sc-jWBwVP sc-dNLxif hwfiyt"
                    for="image"
                  >
                    Profile picture
                  </label>
                  <figure
                    class="sc-kgAjT igvAIP"
                  >
                    <div>
                      <img
                        alt=""
                        src="profile-image.png"
                      />
                    </div>
                    <p>
                      Jake22
                    </p>
                  </figure>
                  <div
                    class="sc-fAjcbJ DOSjX"
                  >
                    <label
                      class="sc-jWBwVP hzoveM"
                      for="fullname"
                    >
                      Full Name
                    </label>
                    <input
                      class="sc-jDwBTQ dCmcys"
                      display="wide"
                      name="fullname"
                      placeholder="Full Name"
                      type="text"
                      value="Jake Tom"
                    />
                  </div>
                  <div
                    class="sc-fAjcbJ DOSjX"
                  >
                    <label
                      class="sc-jWBwVP hzoveM"
                      for="image"
                    >
                      Profile Image
                    </label>
                    <input
                      accept="image/*"
                      class="sc-jDwBTQ dCmcys"
                      display="wide"
                      name="image_url"
                      placeholder="Profile picture"
                      type="file"
                      value=""
                    />
                  </div>
                  <div
                    class="sc-fAjcbJ DOSjX"
                  >
                    <div
                      class="sc-gisBJw crwmrK"
                    >
                      <label
                        class="sc-jWBwVP hzoveM"
                        for="email"
                      >
                        Email
                      </label>
                      <input
                        class="sc-jDwBTQ dCmcys"
                        display="wide"
                        name="email"
                        placeholder="Email"
                        type="text"
                        value="8omemail@google.com"
                      />
                    </div>
                    <div
                      class="sc-gisBJw crwmrK"
                    >
                      <label
                        class="sc-jWBwVP hzoveM"
                        for="username"
                      >
                        Username
                      </label>
                      <input
                        class="sc-jDwBTQ dCmcys"
                        display="wide"
                        name="username"
                        placeholder="Username"
                        type="text"
                        value="Jake22"
                      />
                    </div>
                  </div>
                  <div
                    class="sc-fAjcbJ DOSjX"
                  >
                    <label
                      class="sc-jWBwVP hzoveM"
                      for="bio"
                    >
                      Bio
                    </label>
                    <div
                      class="sc-cHGsZl gHdjpq"
                    >
                      <textarea
                        name="bio"
                        placeholder="bio"
                      >
                        I like coding 
                      </textarea>
                    </div>
                  </div>
                  <div
                    class="sc-fAjcbJ sc-uJMKN tcxiG"
                  >
                    <a
                      href="/dashboard"
                    >
                      <button
                        class="sc-brqgnP fafvla"
                        color="grey"
                        to="/dashboard"
                      >
                        Cancel
                      </button>
                    </a>
                    <button
                      class="sc-brqgnP gyEPEd sc-bbmXgH eUNjaM"
                      color="green"
                      type="submit"
                    >
                      Save Changes
                    </button>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-gzVnrw cAymxl"
        >
          <p
            class="sc-gZMcBi dqwYd"
          >
            International Crafters Â© 2020
          </p>
          <div>
            <a
              class="sc-htoDjs sc-iwsKbI jiENTn"
              href="#"
            >
              Privacy
            </a>
            <a
              class="sc-htoDjs sc-iwsKbI jiENTn"
              href="#"
            >
              Contacts
            </a>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="sc-bdVaJa gOhKpA"
      >
        <div
          class="sc-htpNat dAVxPZ"
        >
          <a
            href="/dashboard"
          >
            <img
              alt="Hackton - Organise hackathons"
              class="sc-ifAKCX gCNDrT"
              src="Hackton-logo.png"
            />
          </a>
          <div
            class="sc-tilXH iUsghC"
          >
            <div
              class="sc-EHOje kXozTT"
            >
              8
              <ul
                class="sc-bZQynM dfAYGq"
              >
                <li>
                  <a
                    href="/dashboard/profile/edit"
                  >
                    Settings
                  </a>
                </li>
                <li>
                  <span>
                    LogOut
                  </span>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-hzDkRC cBfjrB"
      >
        <div
          class="sc-jhAzac jxgZSd sc-jbKcbu iFCIT"
        >
          <div
            class="sc-dVhcbM iWcBSy"
          >
            <h3
              class="sc-jzJRlG dhdOHq"
            >
              Edit Profile
            </h3>
          </div>
          <div
            class="sc-fAjcbJ DOSjX"
          >
            <div
              class="sc-cJSrbW sc-ksYbfQ sc-jqCOkK jnpjEb"
            >
              <form
                action="#"
              >
                <label
                  class="sc-jWBwVP sc-dNLxif hwfiyt"
                  for="image"
                >
                  Profile picture
                </label>
                <figure
                  class="sc-kgAjT igvAIP"
                >
                  <div>
                    <img
                      alt=""
                      src="profile-image.png"
                    />
                  </div>
                  <p>
                    Jake22
                  </p>
                </figure>
                <div
                  class="sc-fAjcbJ DOSjX"
                >
                  <label
                    class="sc-jWBwVP hzoveM"
                    for="fullname"
                  >
                    Full Name
                  </label>
                  <input
                    class="sc-jDwBTQ dCmcys"
                    display="wide"
                    name="fullname"
                    placeholder="Full Name"
                    type="text"
                    value="Jake Tom"
                  />
                </div>
                <div
                  class="sc-fAjcbJ DOSjX"
                >
                  <label
                    class="sc-jWBwVP hzoveM"
                    for="image"
                  >
                    Profile Image
                  </label>
                  <input
                    accept="image/*"
                    class="sc-jDwBTQ dCmcys"
                    display="wide"
                    name="image_url"
                    placeholder="Profile picture"
                    type="file"
                    value=""
                  />
                </div>
                <div
                  class="sc-fAjcbJ DOSjX"
                >
                  <div
                    class="sc-gisBJw crwmrK"
                  >
                    <label
                      class="sc-jWBwVP hzoveM"
                      for="email"
                    >
                      Email
                    </label>
                    <input
                      class="sc-jDwBTQ dCmcys"
                      display="wide"
                      name="email"
                      placeholder="Email"
                      type="text"
                      value="8omemail@google.com"
                    />
                  </div>
                  <div
                    class="sc-gisBJw crwmrK"
                  >
                    <label
                      class="sc-jWBwVP hzoveM"
                      for="username"
                    >
                      Username
                    </label>
                    <input
                      class="sc-jDwBTQ dCmcys"
                      display="wide"
                      name="username"
                      placeholder="Username"
                      type="text"
                      value="Jake22"
                    />
                  </div>
                </div>
                <div
                  class="sc-fAjcbJ DOSjX"
                >
                  <label
                    class="sc-jWBwVP hzoveM"
                    for="bio"
                  >
                    Bio
                  </label>
                  <div
                    class="sc-cHGsZl gHdjpq"
                  >
                    <textarea
                      name="bio"
                      placeholder="bio"
                    >
                      I like coding 
                    </textarea>
                  </div>
                </div>
                <div
                  class="sc-fAjcbJ sc-uJMKN tcxiG"
                >
                  <a
                    href="/dashboard"
                  >
                    <button
                      class="sc-brqgnP fafvla"
                      color="grey"
                      to="/dashboard"
                    >
                      Cancel
                    </button>
                  </a>
                  <button
                    class="sc-brqgnP gyEPEd sc-bbmXgH eUNjaM"
                    color="green"
                    type="submit"
                  >
                    Save Changes
                  </button>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-gzVnrw cAymxl"
      >
        <p
          class="sc-gZMcBi dqwYd"
        >
          International Crafters Â© 2020
        </p>
        <div>
          <a
            class="sc-htoDjs sc-iwsKbI jiENTn"
            href="#"
          >
            Privacy
          </a>
          <a
            class="sc-htoDjs sc-iwsKbI jiENTn"
            href="#"
          >
            Contacts
          </a>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
